src/einsteinpy/tests/test_coordinates/test_coord_transform.py

```diff
@@ -104,6 +104,32 @@ def test_cycle_BLSpherical(bl):
 # Tests for object.__repr__ and object.__str__
 
 
+def test_norm():
+    test_data_x = 1 * u.km
+    test_data_y = 1 * u.km
+    test_data_z = 1 * u.km
+
+    test_obj = Cartesian(x=test_data_x, y=test_data_y, z=test_data_z)
+    assert test_obj.norm() == np.sqrt(3) * u.km
```

Comment:
We should never test with exact values. Tests should be written with some tolerance. This one is passing, but not all will. See [this](https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_allclose.html)

 ```suggestion
     assert_allclose((test_obj.norm()).si.value, (np.sqrt(3) * u.km).si.value, rtol=0, atol=1e-5)
```